examples/interacting_with_js_objects/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[macro_use]
+extern crate neon;
+
+use neon::vm::{Call, JsResult};
+use neon::js::{Variant, JsString, JsObject};
+
+// https://github.com/dfrankland/node-eyeliner/blob/master/native/src/js_object_utils.rs
+pub fn js_value_to_string(js_string: Variant) -> String {
```

Comment:
Similar to the other example, let's get rid of this function and just use `check()` below.